modules/Bio/EnsEMBL/Hive/Scripts/InitPipeline.pm

```diff
@@ -62,7 +62,8 @@ sub init_pipeline {
 
     if($tweaks and @$tweaks) {
         print "> Applying tweaks.\n";
-        $pipeline->apply_tweaks( $tweaks );
+        my ($need_write, $msg_listRef, $json) = $pipeline->apply_tweaks( $tweaks );
```

Comment:
`$msg_listRef` is funny because it mixes snake case and camel case. The whole file is in snake case, so please use `$msg_list_ref`